tests/runner/test_experiment.cfg

```diff
@@ -3,6 +3,7 @@ bparam=True
 iparam=1
 fparam=1.5
 sparam=hello
+lparam = [1, 2, 3]
```

Comment:
can lists contain variables? this was my use case (have a list of featurizers). maybe it would be good to test this?

Reply:
Yes, adding some tests here